CVE-2022-3105.yml

```diff
@@ -37,14 +37,11 @@ fixes:
     commit:
   - note:
     commit:
-  - note: >
-      Taken from NVD references list with Git commit. If you are
-
-      curating, please fact-check that this commit fixes the vulnerability and replace this comment with 'Manually confirmed'
+  - note: Manually confirmed
```

Comment:
I would recommend leaving a short note describing what this commit does, just a few words for the reader to know how it fixes the vulnerability.